quick-start-with-tidb.md

```diff
@@ -269,6 +279,14 @@ aliases: ['/docs-cn/dev/quick-start-with-tidb/','/docs-cn/dev/how-to/get-started
 
 8. 访问集群：
 
+    - 安装 MySQL 客户端。安装过 MySQL 客户端的则可跳过这一步骤。
```

Comment:
```suggestion
    - 安装 MySQL 客户端。如果已安装 MySQL 客户端则可跳过这一步骤。
```

Reply:
Shall we use an ordered list?

Reply:
I am not sure whether the following items are also in sequence.

Reply:
OK. Let's leave it as it is.